lib/srv/ctx.go

```diff
@@ -617,6 +627,25 @@ func (c *ServerContext) Close() error {
 		return trace.Wrap(err)
 	}
 
+	c.mu.Lock()
+	defer c.mu.Unlock()
+
+	// if there was a tty allocated, update the user accounting database
+	// with information that it's now closed
+	if c.termTTY != nil {
+		ttyName, err := utils.TtyName(c.termTTY)
+
```

Comment:
nit: Just as a code style thing, I think we tend to keep error checks immediately next to the code they're checking.

```suggestion
```